scripts/pi-hole/php/header.php

```diff
@@ -60,6 +60,15 @@
         {
             $temperatureunit = $_POST["tempunit"];
         }
+        // Get user-defined temperature limit if set
+        if(isset($setupVars['TEMPERATURELIMIT']))
```

Comment:
It would be more readable to use `TEMPERATURE_LIMIT` as the key.

Reply:
`TEMPERATURELIMIT` was chosen to be similar to the already existing key `TEMPERATUREUNIT`. We can surely change it.